src/views/certifications/views/landing/types.ts

```diff
@@ -1,8 +1,43 @@
 import { RawLandingPageContent } from 'views/certifications/content/schemas/landing-page'
+import { FaqItem } from 'views/certifications/types'
+
+/**
+ * Summaries of each certification program are displayed on the landing page.
+ */
+export interface CertificationProgramSummary {
+	slug: string
+	title: string
+	description: string
+	exams: {
```

Comment:
Is there any overlap here with the program types we can share?

Reply:
Ah yep! I think I'd kind of stubbed these in 😅 updated to grab properties off the `RawCertificationProgram` type, inferred from the certification program schema 👍